src/rules/operator-no-unspaced/index.js

```diff
@@ -159,7 +159,7 @@ export function calculationOperatorSpaceChecker({ root, result, checker }) {
 
       // Media queries
       if (item.name === "media" || item.name === "import") {
-        mediaQueryParser(item.params).walk(node => {
+        mediaQueryParser(item.params).walk("", (node) => {
```

Comment:
Shoot, I really did. Forgot to add a `prepublish` script. Thanks for pointing that out!